packages/stream-metadata/src/riverStreamRpcClient.ts

```diff
@@ -78,7 +84,9 @@ async function mediaContentFromStreamView(
 ): Promise<MediaContent> {
 	const mediaInfo = streamView.mediaContent.info
 	if (mediaInfo) {
-		console.log(`mediaContentFromStreamView: mediaInfo.spaceId=${mediaInfo.spaceId}`)
+		logger.info(`mediaContentFromStreamView`, {
```

Comment:
@tak-hntlabs there's a console.log at line 51

Reply:
fixed. Searched and found the stragglers.